e2e/cypress/integration/integrations/incoming_webhook/slack_formatting_spec.js

```diff
@@ -0,0 +1,325 @@
+            {"short": short, "title": "Short 2", "value": "Another one"}, {"short": true, "title": "Field with link", "value": "<http://example.com|Link>"}], "mrkdwn_in": ["pretext"], "pretext": "Some text here to look at (verify eyes emoji) :eyes:",
+            "text": `${currentID} ${short} This is the text of the attachment. <@${sysadminUser.id}>, there should be a small Jenkins thumbnail off to the right.`, "thumb_url": "https://slack.global.ssl.fastly.net/7bf4/img/services/jenkins-ci_128.png", "title": "A slack attachment", "title_link": "https://www.google.com"
+        }]});
+
+        const testCases = [
+            {short: true, shouldShowShort: true},
+            {short: false, shouldShowShort: false},
+            {short: 'true', shouldShowShort: true},
+            {short: 'false', shouldShowShort: false},
+        ];
+
+        testCases.forEach((testCase, i) => {
+            it('should show table elements based on short value ${i}', () => {
+                const currentID = `${id} - ${i}`;
+                const payload = makePayloadFromShortValue(testCase.short, currentID);
+                cy.postIncomingWebhook({url: incomingWebhook.url, data: payload});
+
+                cy.getLastPost().within((el) => {
+                    cy.get('.attachment__body .post-message__text-container p').should('contain', currentID);
+
+                    if (testCase.shouldShowShort) {
+                        cy.get('table:nth-child(6) > thead > tr > th:nth-child(2)').should('have.text', 'Short 2');
+                        cy.get('table:nth-child(6) > tbody > tr > td:nth-child(2) > p').should('have.text', 'Another one');
+                    } else {
+                        cy.get('table:nth-child(7) > thead > tr > th:nth-child(1)').should('have.text', 'Short 2');
+                        cy.get('table:nth-child(7) > tbody > tr > td:nth-child(1) > p').should('have.text', 'Another one');
+                    }
+                });
+            });
+        });
+    });
+
+    it('MM-T632 Slack compatibility code shouldn\'t mess up characters', () => {
+        const id = 'MM-T632';
+        const text = `${id} <>|<>|`;
+        const payload = {
+            channel: testChannel.name,
+            text,
+        };
+
+        cy.postIncomingWebhook({url: incomingWebhook.url, data: payload});
+
+        cy.getLastPost().within((el) => {
+            cy.get('.post__body p').should('have.text', text);
+        });
+    });
+
+    it('MM-T633 Text in Slack-style attachment is searchable', () => {
```

Comment:
This test failed since "findme" returns nothing when Elasticsearch is disabled. It's [disabled by default](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/fixtures/partial_default_config.json#L424-L426) when running Cypress so it should be enabled explicitly.
With that, I recommend the ff:
- extract this test out into its own spec file at `e2e/cypress/integration/enterprise/integrations/incoming_webhook` since this test specifically requires EE license and we're organizing enterprise tests in `/enterprise` folder.
- add license check to spec file - `cy.apiRequireLicenseForFeature('Elasticsearch')`
- see [enableElasticsearch helper function](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/integration/enterprise/elasticsearch_autocomplete/helpers/index.js#L20) as you'll most likely going to use it.

Reply:
Thanks for updating. I guess this should be removed since it's already moved.

Reply:
Hmm I thought I removed that...